DataConnectors/AWS-S3/CloudWatchLanbdaFunction.py

```diff
@@ -0,0 +1,59 @@
+
+import boto3
+import json
+import csv
+import time
+import pandas as pd
+from datetime import datetime
+
+logs = boto3.client('logs')
+s3 = boto3.resource('s3')
+
+
+# Please set the following parameters:
+log_group_name = "" # Please enter log group name
+log_stream_name = "" # Please enter log stream name
+bucket_name = "" # Please enter bucket name
+bucket_prefix = "" # Please enter bucket prefix that ends with '/' , if no such, leave empty
+output_file_name = "" # Please change to desired name
+start_time = datetime(2022,12,15,6,40) # Please enter start time for exporting logs (year, month, day, hour, minutes) pay attention to time differences, here it should be UTC time
+end_time = datetime(2022,12,15,6,58) # Please enter end time for exporting logs (year, month, day, hour, minutes) pay attention to time differences, here it should be UTC time
+
+def lambda_handler(event, context):
+    unix_start_time = int(time.mktime(start_time.timetuple()))*1000
+    unix_end_time = int(time.mktime(end_time.timetuple()))*1000
+    try:
```

Comment:
Does all the code need to be in the try clause?

Reply:
Only this shouldn't:         df = pd.DataFrame(json_object['events'])
        if df.empty:
            print('No events for specified time')
            return None
        
        # Convert unix time to zulu time for example from 1671086934783 to 2022-12-15T06:48:54.783Z
        df['timestamp'] = pd.to_datetime(df['timestamp'], unit='ms')
        df['timestamp'] = df['timestamp'].dt.strftime('%Y-%m-%dT%H:%M:%S.%f').str[:-3]+'Z'

The rest should so I decided to leave it in the same try clause